test/sparsity/test_marlin.py

```diff
@@ -0,0 +1,132 @@
+import torch
+import copy
+import pytest
+
+from torch import nn
+from torch.testing._internal.common_utils import TestCase, run_tests
+from torchao.dtypes import MarlinSparseLayoutType
+from torchao.sparsity.sparse_api import apply_fake_sparsity
+from torchao.quantization.quant_api import int4_weight_only, quantize_
+from torchao.sparsity.marlin import (
+    pack_to_marlin_24,
+    unpack_from_marlin_24,
+    inject_24
+)
+from torchao.quantization.utils import (
+    get_group_qparams_symmetric,
+    groupwise_affine_quantize_tensor_from_qparams,
+)
+
+
+class SparseMarlin24(TestCase):
+
+    @pytest.mark.skipif(not torch.cuda.is_available(), reason="Need CUDA available")
+    def test_quant_sparse_marlin_layout_eager(self):
+        torch.manual_seed(0)
```

Comment:
nit: maybe this can be moved to setup like in https://github.com/pytorch/pytorch/blob/dbeb8a1691a9dba4bf2a10566fd55705d67c24a9/torch/testing/_internal/common_quantization.py#L533

Reply:
Refactored the test to use the `setUp` method